front/components/poke/subscriptions/table.tsx

```diff
@@ -46,6 +46,7 @@ function prepareSubscriptionsForDisplay(
       id: s.sId ?? "unknown",
       name: s.plan.code,
       status: s.status,
+      stripeSubscriptionId: s.stripeSubscriptionId,
```

Comment:
When null it displays just an empty cell?

Reply:
Yes 👍